Tools-Override/codeOptimization.targets

```diff
@@ -81,7 +81,8 @@
   </Target>
 
   <!-- We need the OptimizationData package in order to be able to optimize the assembly -->
-  <Target Name="RestoreOptimizationDataPackage" BeforeTargets="CoreCompile"
+  <Target Name="RestoreOptimizationDataPackage"
```

Comment:
Hmm, I can't think of one. Created https://github.com/dotnet/buildtools/issues/1436.